.github/workflows/frameworks.yml

```diff
@@ -538,8 +538,7 @@ jobs:
 
   lightgbm_integration_tests:
     needs: diff
-    #if: ${{ (github.event_name == 'pull_request' && needs.diff.outputs.lightgbm == 'true') || github.event_name == 'push' }}
-    if: ${{ false }}
+    if: ${{ (github.event_name == 'pull_request' && needs.diff.outputs.lightgbm == 'true') || github.event_name == 'push' }}
```

Comment:
I wasn't sure if this change is good or not, but the xgboost entry has it and I've been following xgboost as an example.

Reply:
This is fine! We just disabled all the tests for old frameworks by setting `if false`.